### ISSUE TITLE
What are those? (Log provided)

### ISSUE THREAD

**Issue by jmytch** [seud0nym/tch-gui-unhide#37]
Keep getting those spams - Failed to send to ff02::1%lan@br-lan (Permission denied)
Uploaded a log to see if there is anything to help stop that spam in the log viewer, it was getting on my nerves :( 
[log.txt](https://github.com/seud0nym/tch-gui-unhide/files/6872760/log.txt)

**Comment by seud0nym** [comment]
Can you also put up /etc/config/network? (Remove any public IP addresses if any).

**Comment by jmytch** [comment]
config` interface 'loopback'
	option ifname 'lo'
	option proto 'static'
	option ipaddr '127.0.0.1'
	option netmask '255.0.0.0'

config globals 'globals'
	option default_ps '0'
	option ula_prefix 'fda2:xxxx:xxxx::/48'

config interface 'lan'
	option type 'bridge'
	option proto 'static'
	option netmask '255.255.255.0'
	option ip6hint '0'
	option ipaddr '192.168.0.1'
	option force_link '1'
	option rpfilter '1'
	list pppoerelay ''
	option tch_ip6assign '64'
	list ifname 'eth0'
	list ifname 'eth1'
	list ifname 'eth2'
	list ifname 'eth3'
	option ip6assign '64'
	option ipv6 '1'

config switch
	option name 'bcmsw'
	option reset '1'
	option enable_vlan '0'

config interface 'ppp'
	option proto 'pppoe'
	option metric '10'
	option keepalive '4,20'
	option iface6rd '0'
	option graceful_restart '1'
	option auto '0'
	option password 'xxxxxxxxx'
	option username '1xxxxxxxxxxx6'

config interface 'ipoe'
	option proto 'dhcp'
	option metric '1'
	option reqopts '1 3 6 43 51 58 59'
	option release '1'
	option iface6rd '0'
	option vendorid 'technicolor'
	option ifname 'ptm0'
	option auto '1'

config interface 'wan6'
	option defaultreqopts '0'
	option reqopts '23 17'
	option reqaddress 'force'
	option noslaaconly '1'
	option iface_464xlat '0'
	option forceprefix '1'
	option soltimeout '240'
	option peerdns '1'
	option proto 'dhcpv6'
	option reqprefix 'auto'
	option ifname '@wan'
	option auto '0'

config interface 'wwan'
	option proto 'mobiled'
	option session_id '0'
	option iface_464xlat '1'
	option enabled '0'
	option profile '1'

config device 'eth4'
	option name 'eth4'
	option mtu '1500'
	option mtu6 '1500'
	option neighreachabletime '1200000'
	option neighgcstaletime '2400'

config device 'atm_8_35'
	option name 'atm_8_35'
	option neighreachabletime '1200000'
	option neighgcstaletime '2400'

config device 'ptm0'
	option name 'ptm0'
	option neighreachabletime '1200000'
	option neighgcstaletime '2400'

config ppp_placeholder
	option uciname 'pppoe-wan'

config config 'config'
	option wan_mode 'pppoe'

config interface 'wan'
	option keepalive '4,20'
	option ifname 'ptm0'
	option iface6rd '0'
	option metric '10'
	option graceful_restart '1'
	option auto '1'
	option proto 'pppoe'
	option username '1xxxxxxxxxx6'
	option peerdns '1'
	option password 'xxxxxxxx'
	option ipv6 '0'

**Comment by seud0nym** [comment]
So you have got IPv6 turned off everywhere except the LAN? No IPv6 from your RSP?

According to this post: https://github.com/openwrt/odhcp6c/issues/55#issuecomment-343939681

> this trace is often triggered when there's no valid link local address in use or when no layer3 connectity is supported on the device eg due to the device being a bridge member
Can you display the output of "ip -6 addr show" to confirm you've valid link local addresses in use ?

**Comment by jmytch** [comment]
Yes, I churned to another RSP recently to take up the offer promotion of 100 Mbps / 40 Mbps

This RSP dont do IPv6 but I still have IPv6 enabled for LAN side. Am I supposed to turn IPv6 off for LAN too if RSP don't do IPv6?

**Comment by seud0nym** [comment]
I don't think it should be necessary, but it is worth a try.

Try that first, and if it makes no difference, maybe delete the `option ula_prefix 'fda2:xxxx:xxxx::/48'` line and then `/etc/init.d/network restart`
